components/validator/issuer.go

```diff
@@ -0,0 +1,66 @@
+package validator
+
+import (
+	"context"
+	"time"
+
+	"github.com/iotaledger/iota-core/pkg/blockfactory"
+	"github.com/iotaledger/iota-core/pkg/model"
+	iotago "github.com/iotaledger/iota.go/v4"
+)
+
+func issueValidatorBlock(ctx context.Context) {
+	blockIssuingTime := time.Now()
+	nextBroadcast := blockIssuingTime.Add(ParamsValidator.CommitteeBroadcastInterval)
+
+	// Use 'defer' because nextBroadcast is updated during function execution, and the value at the end needs to be used.
+	defer func() {
+		executor.ExecuteAt(accountID, func() { issueValidatorBlock(ctx) }, nextBroadcast)
+	}()
+
+	if !ParamsValidator.IgnoreBootstrapped && !deps.Protocol.MainEngineInstance().IsBootstrapped() {
+		Component.LogDebug("Not issuing validator block because node is not bootstrapped yet.")
+
+		return
+	}
+
+	var modelBlock *model.Block
+	if deps.Protocol.MainEngineInstance().SybilProtection.SeatManager().Committee(deps.Protocol.CurrentAPI().TimeProvider().SlotFromTime(blockIssuingTime)).HasAccount(accountID) {
+		var err error
+		modelBlock, err = deps.BlockIssuer.CreateValidationBlock(ctx,
+			blockfactory.WithIssuingTime(blockIssuingTime),
+			blockfactory.WithPayload(&iotago.TaggedData{
+				Tag: []byte("VALIDATOR BLOCK"),
+			}),
+		)
+		if err != nil {
+			Component.LogWarnf("error creating committee validator block: %s", err.Error())
+
+			return
+		}
+	} else {
+		var err error
+		modelBlock, err = deps.BlockIssuer.CreateBlock(ctx,
```

Comment:
We should be issuing validator blocks, so we can track performance just the same, but with a lower emission frequency.

Reply:
Done. The only thing is that when implementing tip selection for validator blocks, we should keep track of validator blocks from current committee members.